apps/worker/src/app/telemetry/utils/sendDataToMixpanel.utils.ts

```diff
@@ -0,0 +1,22 @@
+import { HttpService } from '@nestjs/axios';
+import { Logger } from '@nestjs/common';
+import { firstValueFrom } from 'rxjs';
+
+export async function sendDataToMixpanel(httpService: HttpService, event: string, properties: any) {
+  try {
+    Logger.debug(`Sending '${event}' event to Mixpanel`);
+    const dataToSend = {
+      event,
+      properties: {
+        ...properties,
+        timestamp: new Date().toISOString(),
+      },
+    };
+
+    const res = await firstValueFrom(httpService.post(process.env.OS_TELEMETRY_URL as string, dataToSend));
+    Logger.debug(`Response from Mixpanel: ${JSON.stringify(res.data)}`);
+    Logger.debug(`'${event}' event sent to Mixpanel successfully.`);
```

Comment:
```suggestion
    Logger.debug(`'${event}' event sent to trace.novu.sh successfully.`);
```

Reply:
I did not wanted to give away the domain name. So people dont start spamming thats was the whole reason I used it from env variable instead of the hardcoding. I'll remove this log altogether